index.html

```diff
@@ -0,0 +1,1005 @@
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:wght@700&family=Poppins:wght@200;400;700;800&display=swap">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+
+    <script>
+        window.smartlook || (function(d) {
+            var o = smartlook = function() {
+                    o.api.push(arguments)
+                },
+                h = d.getElementsByTagName('head')[0];
+            var c = d.createElement('script');
+            o.api = new Array();
+            c.async = true;
+            c.type = 'text/javascript';
+            c.charset = 'utf-8';
+            c.src = 'https://web-sdk.smartlook.com/recorder.js';
+            h.appendChild(c);
+        })(document);
+        smartlook('init', '3d8cc3fad36fd64fa8461c440815b246c3b75961', {
+            region: 'eu'
+        });
+    </script>
+</head>
+
+<body>
+    <nav class="container-fluid">
+        <div class="container-fluid">
+            <div class="d-flex flex-wrap align-items-center justify-content-center justify-content-lg-start">
+                <a href="/" class="d-flex align-items-center mb-2 mb-lg-0 text-white text-decoration-none">
+                    <div class="logo">logo</div>
+                </a>
+                <a class="navbar-brand" href="#">Navbar</a>
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <ul class="nav col-12 col-lg-auto me-lg-auto mb-2 justify-content-center mb-md-0">
+                    <li><a href="#" class="nav-link px-2 text-secondary">Features</a></li>
+                    <li><a href="#" class="nav-link px-2 text-white">Pricing</a></li>
+                    <li><a href="#" class="nav-link px-2 text-white">Industries</a></li>
+                </ul>
+                <!--
+                <form class="col-12 col-lg-auto mb-3 mb-lg-0 me-lg-3" role="search">
+                <input type="search" class="form-control form-control-dark text-bg-dark" placeholder="Search..." aria-label="Search">
+                </form>-->
+
+                <div class="text-end">
+                    <!--<button type="button" class="btn btn-outline-light me-2">Login</button>-->
+                    <button type="button" class="btn btn-info">Request a meeting</button>
+                </div>
+            </div>
+        </div>
+    </nav>
+
+    <header class="container-fluid message">
+        <div class="row">
+            <div class="col-12 text-center">
+                <p><i><b>FlyArrow.io</b></i> is a project of the <b>ARROW DAO</b>, a non-profit decentralized organization building community owned aerial solutions. <a href="https://www.arrowair.com" target="_blank">Visit the DAO website</a></p>
```

Comment:
How about:
`FlyArrow.io is the innovative software development project of ARROW - a DAO committed to advancing planet-friendly air mobility as a fundamental human right. Discover the boundless potential of a better future, one flight at a time.`

The 'non-profit' part was mainly added because not everyone knows what a DAO is, but they can form an idea about a 'non-profit' organization. I'm not too fuzzed about using the word, as Arrow DAO -is- non-profit.